From f6162ced588c17c1141a8f6154475a299a2ebc0d Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Tue, 26 Apr 2022 22:35:35 +0200
Subject: [PATCH] Makefile: quote CC

It can contain command line options, and therefore spaces, and so
needs to be quoted.

Upstream-Status: Submitted [https://github.com/ColinIanKing/stress-ng/pull/188]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 Makefile        | 2 +-
 Makefile.config | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 501093f5..8a454edd 100644
--- a/Makefile
+++ b/Makefile
@@ -432,7 +432,7 @@ stress-ng: $(OBJS)
 	$(V)sync
 
 config.h:
-	$(MAKE) CC=$(CC) STATIC=$(STATIC) -f Makefile.config
+	$(MAKE) CC="$(CC)" STATIC=$(STATIC) -f Makefile.config
 
 .PHONY:
 makeconfig: config.h
diff --git a/Makefile.config b/Makefile.config
index b76d0903..f03f7a85 100644
--- a/Makefile.config
+++ b/Makefile.config
@@ -69,7 +69,7 @@ else
 endif
 
 
-MAKE_OPTS=CC=$(CC) -f Makefile.config --no-print-directory
+MAKE_OPTS=CC="$(CC)" -f Makefile.config --no-print-directory
 
 comma = ,
 
